src/main/java/com/dev/cinema/service/mapper/UserMapper.java

```diff
@@ -3,11 +3,20 @@
 import com.dev.cinema.model.User;
 import com.dev.cinema.model.dto.request.UserRequestDto;
 import com.dev.cinema.model.dto.response.UserResponseDto;
+import com.dev.cinema.service.RoleService;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
 @Service
 public class UserMapper implements MapperToDto<User,
         UserResponseDto>, MapperToEntity<User, UserRequestDto> {
+    private final RoleService roleService;
+
+    @Autowired
+    public UserMapper(RoleService roleService) {
+        this.roleService = roleService;
+    }
+
```

Comment:
Why are you add this?

Reply:
I thought I will use it and forgot to delete it